UnityAds/src/main/java/com/mopub/mobileads/UnityInterstitial.java

```diff
@@ -128,28 +130,16 @@ protected void show() {
             return;
         }
 
-        if (UnityAds.isReady(mPlacementId)) {
-            // Lets Unity Ads know when ads succeeds to show
-            MediationMetaData metadata = new MediationMetaData(mContext);
-            metadata.setOrdinal(++impressionOrdinal);
-            metadata.commit();
-
+        if (mLoadedPlacementId == null) {
```

Comment:
We explicitly handle this case from our SDK, and may be valuable to root up to us to understand if it ever occurs, so removing this check in favor of the `null` check in the Webview could be valuable

Reply:
So i have to distinguish between when the ad was not loaded and when the actual placement id is null.  That is what this check is doing.  In the event its null, i use the created placement id (what we would have attempted to load with) as the placement id.  in the event that the user supplied a null placement mPlacementId would also be null and that would be sent to webview.

Reply:
Our SDK should be able to accept this, and would allow us to track *if* this was ever actually occurring. If we fail here due to `null` placementId, then we will never know about it from our side

Reply:
so if you look im always calling show, im just swapping which value im using as the placement id so I don't send null for unloaded placementIds when we know which placement id this is for.

Reply:
Ah I see now. Thanks for the context.

I don't believe that this behavior would be preferred. It could lead to fill invalidation behavior where this "default" placement ID is loaded in a separate waterfall and then fails to show in another spot

Reply:
resolved offline.

Reply:
Just adding that this logic seems to differ from the iOS adapter: https://github.com/Applifier/mopub-ios-mediation/blob/b79b0c60606adc95bbb920a1e86293ee1ddf5b2f/UnityAds/UnityAdsInterstitialCustomEvent.m#L97